POEApi.Model/Data.xml

```diff
@@ -634,6 +635,7 @@
       <Item name="Sambar Sceptre"/>
       <Item name="Behemoth Mace"/>
       <Item name="Coronal Maul"/>
+      <Item name="Morning Star"/>
```

Comment:
I can see the morning star getting added here but I don't see a supporting unit test for either of them :/